### PR TITLE
doc(examples): add examples without macro usage

### LINKED ISSUE #266: simple example without macros
Could you add two simple examples with handler functions instead of `middleware!{}` and `router!{}`? Like `server.get("foo/bar", my_foo_bar_handler)`?

### PR DESCRIPTION
**Pull request by Ryman**
Closes #266.

### CODE REVIEW

**Comment by Ryman** [comment]
@flosse Does this satisfy or would you like something else added?

**Comment by flosse** [comment]
@Ryman yes, that's what I was looking for :) thanks!

File: examples/hello_world.rs [hunk 1]
```diff
@@ -1,9 +1,13 @@
 #[macro_use] extern crate nickel;
 
-use nickel::{Nickel, HttpRouter};
+use nickel::{Nickel, HttpRouter, Request, Response, MiddlewareResult};
+
+fn hello_world<'mw>(_req: &mut Request, res: Response<'mw>) -> MiddlewareResult<'mw> {
```
Comment (badboy) [inline examples/hello_world.rs]: This currently doesn't work because `MiddlewareResult` also expects a type like `()` to make it work.

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were style issues with your Pull Request:

 * Commit: cd773d791c28d345402a1bcb7c79169a0a20bb91
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/nickel-org/nickel.rs/blob/master/contributing.md

---
This message was auto-generated by https://gitcop.com

File: examples/hello_world.rs (hunk 1 above)
Comment (Ryman) [inline examples/hello_world.rs]: @badboy Thanks, just pushed a fix!

**Comment by Ryman** [comment]
@homu r+

**Pull request merged**